src/voprf.rs

```diff
@@ -191,16 +191,16 @@ impl<G: Group, H: BlockInput + Digest> NonVerifiableClient<G, H> {
         }
     }
 
-    #[cfg(test)]
+    #[cfg(any(feature = "internal", test))]
     /// Only used for test functions
     pub fn get_blind(&self) -> <G as Group>::Scalar {
         self.blind
     }
 
-    #[cfg(test)]
+    #[cfg(any(feature = "internal", test))]
     /// Only used for testing zeroize
     pub fn as_ptrs(&self) -> Vec<Vec<u8>> {
-        vec![
+        alloc::vec![
             self.data.clone(),
             <G as Group>::scalar_as_bytes(self.blind).to_vec(),
```

Comment:
I noticed that `as_ptrs()` is pretty much covered by `serialize()`, except that the output is `Vec<Vec<u8>>` instead of `Vec<u8>`, is there any reason for this?
I think, if in any case we keep these functions around, they should probably be renamed, `as_ptr` is usually used in Rust to return a `*const x`.

Reply:
Great point. I initially had as_ptr() for some of these functions and was returning an actual pointer, with the intention of testing "zeroize on drop" functionality. But, dropping in tests turned out to be somewhat finicky, so I downgraded those tests to just test zeroize properly without dropping.

However, you make the excellent point that if we are just testing zeroize, we don't need pointers, and so just relying on the `serialize()` functions is perfect! Thanks!